nixos/modules/security/pam.nix

```
@@ -1068,6 +1082,57 @@ in
       };
     };
 
+    security.pam.rssh = {
+      enable = lib.mkEnableOption ''
+        authenticating using a signature performed by the ssh-agent.
+      '';
```

nit: Currently, this renders weirdly in `man configuration.nix`:

```
       security.pam.rssh.enable
           Whether to enable authenticating using a signature performed by the ssh-agent. .
```

```suggestion
      enable = lib.mkEnableOption "authenticating using a signature performed by the ssh-agent";
```